reference_files/1.0.0/complex.yaml

```diff
@@ -3,190 +3,190 @@
 %YAML 1.1
 %TAG ! tag:stsci.edu:asdf/
 --- !core/asdf-1.0.0
-asdf_library: !core/software-1.0.0 {author: Space Telescope Science Institute, homepage: 'http://github.com/spacetelescope/asdf',
-  name: asdf, version: 1.0.0}
+asdf_library: !core/software-1.0.0 {author: The ASDF Developers, homepage: 'http://github.com/asdf-format/asdf',
+  name: asdf, version: 3.0.1}
 datatype<c16: !core/ndarray-1.0.0
   data: [!core/complex-1.0.0 0j, !core/complex-1.0.0 0j, !core/complex-1.0.0 (nan+nanj),
-    !core/complex-1.0.0 (nan+infj), !core/complex-1.0.0 (nan-infj), !core/complex-1.0.0 -1.79769313486e+308j,
-    !core/complex-1.0.0 1.79769313486e+308j, !core/complex-1.0.0 2.22044604925e-16j,
-    !core/complex-1.0.0 1.11022302463e-16j, !core/complex-1.0.0 2.22507385851e-308j,
+    !core/complex-1.0.0 (nan+infj), !core/complex-1.0.0 (nan-infj), !core/complex-1.0.0 -1.7976931348623157e+308j,
```

Comment:
The values in the existing `complex.yaml` are not correct for a 64 bit float (real or imaginary portion of a complex 128, yaml will treat all numbers as 64 bit so the expanded representation should be seen for all  bit-depths of complex values in this file).
```python
>> np.finfo(np.float64).max == 1.79769313486e+308
False
>> np.finfo(np.float64).max == 1.7976931348623157e+308
True
```